docs/standard/index.md

```diff
@@ -14,65 +14,48 @@ ms.assetid: bbfe6465-329d-4982-869d-472e7ef85d93
+## Porting .NET Framework Code to .NET Core
 
-**Type Safety** -- The primary .NET model for functionality and memory representation is "types". Types define shape and optionally behavior. The runtime ensures that calling code can only operate on types according to their definition and specified visibility of members, providing consistent, reliable and secure results.
+If you're looking to port an application, service, or some component of a system to .NET Core, check out [Porting to .NET Core from .NET Framework](../core/porting/index.md).
 
-Features
---------
+## Porting a NuGet package from to .NET Standard
 
-**User-defined Value Types** -- Value types are a useful category of types since they offer the semantic of "pass by value" instead of "pass by reference", as is the case for classes. Value types are most obviously usefully for numeric data. .NET enables value types for both primitive types, like integers, and user-defined types.
+If you're looking to port a NuGet package to .NET Standard, check out [Porting to .NET Core from .NET Framework](../core/porting/index.md).
 
-**Generic types** -- Generic types are types with one or more type parameters that can be specified on a per-instantiation basis. This is useful for many types, which otherwise would  expose contents as the Object type or require multiple type definitions. For example, a given instantiation of a collection type can be made specific to People, GPS locations or strings.
+## Interested in .NET Fundamentals
 
-**Reflection** -- .NET defines a metadata format that describes the types within a binary. The reflection subsystem uses this data, exposing APIs for both reading and instantiating types at runtime. This facility is very useful for dynamic scenarios where it is not convenient to know the exact implementation of a program ahead of time.
+If you're interested in some of the fundamental concepts of .NET, check out:
 
-**Flexible code generation** -- .NET does not prescribe a specific approach to transforming .NET binaries into machine code. Many approaches have been used successfully, including interpretation, just-in-time (JIT) compilation, ahead-of-time (AOT) compilation with JIT fallback and AOT compilation with no JIT fallback. Each of these strategies can be valuable and there are opportunities for using them together.
+* [.NET Architectural Components](components.md)
+* [.NET Standard Library](library.md)
+* [C# Guide](../csharp/index.md)
+* [F# Guide](../fsharp/index.md)
+* [Visual Basic Guide](../visual-basic/index.md)
+* [Native Interoperability](native-interop.md)
+* [Garbage Collection](garbagecollection/index.md)
+* [Base Types in .NET](base-types/index.md)
+* [Collections](collections/index.md)
+* [Dates, times, and time zones](datetime/index.md)
+* [Asynchronous Programming](async.md)
```

Comment:
I decided to split out the languages from here and just call them Major concepts.